libsrtp_rs/include/libsrtp_rs/libsrtp_rs.h

```diff
@@ -0,0 +1 @@
+void rust_noop();
```

Comment:
Was this hand generated or with cbindgen? I see you depend on cbindgen, but I don't see it used anywhere.

Reply:
This was hand-generated.  I had something weird with my rust setup that caused cbindgen not to work, so I hand-wrote something to get it going and forgot to remove the dependency.